vscode-extension/package.json

```diff
@@ -33,5 +33,17 @@
     "tslint": "^5.16.0",
     "typescript": "^3.6.4",
     "vscode": "^1.1.35"
+  },
+  "contributes": {
+    "configuration": {
+      "title": "clj-kondo",
+      "properties": {
+        "clj-kondo.config.configArgs": {
```

Comment:
Maybe just call it `clj-kondo.config`, `configArgs` seems a bit redundant to me.